src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -106,9 +106,12 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
 
   @volatile private var sessionUsed = false
 
+  private val metricsCategory = s"cassandra.readjournal.${system.name}.hash.${System.identityHashCode(this)}"
```

Comment:
we should use the plugin config path here also, but that is more problematic, since it's not passed by Akka. I think we should change Akka to optionally support constructor with a third parameter passing the configPath.

`system: ExtendedActorSystem, config: Config, configPath: String`

For now, you can hardcode this to "cassandra-query-journal", i.e. the default config path.